src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/searchablelist/SearchableListStyles.kt

```diff
@@ -13,11 +16,18 @@ class SearchableListStyles : Stylesheet() {
         val searchField by cssclass("wa-search-field")
         val searchListView by cssclass("wa-search-list-view")
         val icon by cssclass("wa-searchable-list-icon")
+        val separator by cssclass()
 
         fun searchIcon(size: String = "1em") = MaterialIconView(MaterialIcon.SEARCH, size)
     }
 
     init {
+
+        separator {
+            padding = box(15.px)
+            backgroundColor += Color.TRANSPARENT
+        }
```

Comment:
Can get rid of these styles if using the Vbox `spacing` attribute as mentioned above